pkgs/development/interpreters/python/hooks/conda-unpack-hook.sh

```diff
@@ -0,0 +1,18 @@
+# Setup hook to use in case a conda binary package is fetched
+echo "Sourcing conda unpack hook"
+
+condaUnpackPhase(){
+    echo "Executing condaUnpackPhase"
+    runHook preUnpack
+
+    # use lbzip2 for parallel decompression (bz2 is slow)
+    lbzip2 -dc -n $NIX_BUILD_CORES $src | tar --exclude='info' -x
+
+#     runHook postUnpack # Calls find...?
```

Comment:
```suggestion
    # runHook postUnpack # Calls find...?
```

Reply:
I copied this line from another python hook. I'd really like to get rid of this completely. I think we should not disable postUnpack but instead fix the problem it is causing. But If I fix this here, it will cause a mass rebuild, so I better keep this for another PR. Though, I will fix the indentation as you proposed.